Disable test for docstrings in fortran extension modules, as it fails due to an error
in autogenerated test code (int * versus npy_intp*).

Author: Jakob Schiotz (Techn. Univ. Denmark) <schiotz@fysik.dtu.dk>

--- numpy/f2py/tests/test_module_doc.old	2023-10-23 15:49:00.858332420 +0200
+++ numpy/f2py/tests/test_module_doc.py	2023-10-23 15:50:04.072826243 +0200
@@ -7,7 +7,7 @@
 from numpy.testing import IS_PYPY
 
 
-class TestModuleDocString(util.F2PyTest):
+class Disabled_for_intel_TestModuleDocString(util.F2PyTest):
     sources = [
         util.getpath("tests", "src", "module_data",
                      "module_data_docstring.f90")
